content/ai_exchange/content/docs/4_runtime_application_security_threats.md

```diff
@@ -21,10 +21,10 @@ Note: some controls in this document are application security controls that are
   Useful standards include:
   - See [OpenCRE on technical application security controls](https://www.opencre.org/cre/636-660)
   - The ISO 27002 controls only partly cover technical application security controls, and on a high abstraction level
-  - More detailed and comprehensive control overviews can be found in for example Common criteria protection profiles (ISO/IEC 15408 with evaluation described in ISO 18045),
+  - More detailed and comprehensive control overviews can be found in for example, Common criteria protection profiles (ISO/IEC 15408 with evaluation described in ISO 18045),
   - or in [OWASP ASVS](https://owasp.org/www-project-application-security-verification-standard/)
 - Operational security  
-  When models are hosted by third parties then security configuration of those services deserves special attention. Part of this configuration is [model access control](/goto/modelaccesscontrol/):  an important mitigation for security risks. Cloud AI configuration options deserve scrutiny, like for example opting out when necessary of monitoring by the third party - which could increase the risk of exposing sensitive data.
+  When models are hosted by third parties then security configuration of those services deserves special attention. Part of this configuration is [model access control](/goto/modelaccesscontrol/):  an important mitigation for security risks. Cloud AI configuration options deserve scrutiny, like for example opting out of third party monitoring when necessary  - which could increase the risk of exposing sensitive data.
```

Comment:
please revert, this is not a third party monitoring what you do, this is a third party monitoring what you do at the third party